release/models/optical-transport/openconfig-wavelength-router.yang

```diff
@@ -554,6 +583,20 @@ module openconfig-wavelength-router {
         is configured";
     }
 
+    leaf ase-recovery-hysteresis {
+      type decimal64 {
+        fraction-digits 2;
+      }
+      units dB;
+      description
+        "Defines the optical power offset value relative to the
+        ASE injection threshold at which point ASE should be removed
+        and the media-channel should be restored.
+        Also specifies the optical power offset relative to
```

Comment:
I wasn't sure I understood this second sentence as well. It seems the "recovery" hysteresis would control the initial addition of the media channel as well? In that sense is recovery the right name here? I was looking back at the doc when we were discussing this and it looks like originally you were thinking of calling this "ase-injection-hysteresis" although I agree in changing it as I'm not clear that is the right name here either.

Apologies, I could definitely be misunderstanding, but I think this is about when ASE should be removed and the media channel added. Would something like "ase-removal-hysteresis" be more explicit? Or flipping things around since we already have "ase-injection-threshold" would "media-channel-injection-hysteresis" fit well?

Reply:
To Darren's point below, maybe instead of hysteresis "offset" could be an alternative?

Reply:
@dplore @ejbrever thanks for the review and comments. I tried to clarify this and liked the idea of calling this "media-channel-injection-hysteresis" instead, but it made sense to me to remove hysteresis and use "offset" instead.
Please take a look at the updated name and description and we can adjust further if needed.

Reply:
Looks good, thanks.